src/main/java/com/exasol/adapter/document/documentnode/json/JsonNodeFactory.java

```diff
@@ -48,7 +48,8 @@ public static JsonNodeFactory getInstance() {
         case NUMBER:
             return new JsonNumberNode((JsonNumber) jsonValue);
         default:
-            throw new UnsupportedOperationException("Unsupported json type: " + jsonValue.getValueType());
+            throw new UnsupportedOperationException(
+                    "F-VSDF-4 Unsupported json type: " + jsonValue.getValueType() + ". Please open an issue.");
```

Comment:
What is the difference between `F` and `E` prefixes? Fail and error?

Reply:
Yes. It's documented here:
https://intranet.srv.exasol.com/display/RD/Unified+error+handling
Section: Severity

Reply:
Thanks, just saw the email from Sebastian also.